pkgs/by-name/dm/dmt-core/package.nix

```diff
@@ -0,0 +1,71 @@
+{ lib, buildPythonApplication, fetchurl, fetchPypi, pythonPackages }:
```

Comment:
Why was this switched to `pythonPackages`? That's going to be python 2.x, which is wrong.

Reply:
also, `buildPythonApplication` comes from `python3Packages`, so shouldn't be passed in as a separate arg.

Reply:
> Why was this switched to `pythonPackages`? That's going to be python 2.x, which is wrong.

It's in the PR. I was told too but I now know it's python 2.x so definitely won't be doing that.